plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -90,7 +90,34 @@ protected function initialize() {
 		parent::initialize();
 		add_action( 'wp_loaded', array( $this, 'register_empty_cart_message_block_pattern' ) );
 		add_action( 'wp_print_footer_scripts', array( $this, 'print_lazy_load_scripts' ), 2 );
-		add_filter( 'hooked_block_types', array( $this, 'register_hooked_block' ), 10, 4 );
+		/**
+		 * The hooked_block_{$hooked_block_type} filter was added in WordPress 6.5.
+		 * We are the only code adding the filter 'hooked_block_woocommerce/mini-cart'.
+		 * Using has_filter() for a compatibility check won't work because add_filter() is used in the same file.
+		 */
+		if ( version_compare( get_bloginfo( 'version' ), '6.5', '>=' ) ) {
+			add_filter( 'hooked_block_woocommerce/mini-cart', array( $this, 'modify_hooked_block_attributes' ), 10, 5 );
+		}
+		add_filter( 'hooked_block_types', array( $this, 'register_hooked_block' ), 9, 4 );
+	}
+
+	/**
+	 * Callback for the Block Hooks API to modify the attributes of the hooked block.
+	 *
+	 * @param array|null                      $parsed_hooked_block The parsed block array for the given hooked block type, or null to suppress the block.
+	 * @param string                          $hooked_block_type   The hooked block type name.
+	 * @param string                          $relative_position   The relative position of the hooked block.
+	 * @param array                           $parsed_anchor_block The anchor block, in parsed block array format.
+	 * @param WP_Block_Template|WP_Post|array $context             The block template, template part, `wp_navigation` post type,
+	 *                                                             or pattern that the anchor block belongs to.
+	 * @return array|null
+	 */
+	public function modify_hooked_block_attributes( $parsed_hooked_block, $hooked_block_type, $relative_position, $parsed_anchor_block, $context ) {
+		$block_styles = wp_get_global_styles( array( 'blocks', 'core/navigation', 'typography', 'fontSize' ) );
+
+		$parsed_hooked_block['attrs']['style']['typography']['fontSize'] = $block_styles;
```

Comment:
I tested this and it doesn't seem to be a real issue. However, before setting the font size, we should ensure it's not already set by checking `['typography']['fontSize']` and `wp_get_global_styles(['blocks', 'woocommerce/mini-cart', 'typography', 'fontSize'])`: what do you think?

The same applies to the Customer Account block.

Reply:
Done -> [a8fd0cd](https://github.com/woocommerce/woocommerce/pull/53583/commits/a8fd0cd18966a057d83842c177eb98e34bdd65a8)

I don't think we should check the `wp_get_global_styles` as well since whatever the value is returned is gonna be a valid value to set as a fontSize for the mini cart block, of course, if it does not have any value already set before.

Reply:
Why? If `wp_get_global_styles(['blocks', 'woocommerce/mini-cart', 'typography', 'fontSize'])` is defined, we should not set to the blocks the style of the Navigation block, right?

Reply:
> It looks like calling `wp_get_global_styles(['blocks', 'woocommerce/mini-cart', 'typography', 'fontSize'])` will always return a value since it considers styles inherited by the mini-cart so if we check for it the condition will always prevent us from assigning a new value to the fontSize.

I did a quick test and I noticed that when the `mini-cart` is defined in the theme.json, `wp_get_global_styles(['blocks', 'woocommerce/mini-cart', 'typography', 'fontSize']` returns a string like:
```
 var(--wp--preset--font-size--medium)
```
If not, it returns the entire style object.

The same check should happen for `wp_get_global_styles( array( 'blocks', 'core/navigation', 'typography', 'fontSize' )`. Not all the block themes customize to the navigation block.

Reply:
done -> [763ecbf](https://github.com/woocommerce/woocommerce/pull/53583/commits/763ecbf185499bf264f3ced32eec65ff58a05e4f)